<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

     <title>L. Thibério Rangel</title>

     <style type="text/css">
        * { margin: 0; padding: 0; }
        body { font: 16px Helvetica, Sans-Serif; line-height: 24px; background: url(images/noise.jpg); }
        .clear { clear: both; }
        #page-wrap { width: 1000px; margin: 40px auto 60px; }
        #pic { float: right; margin: -30px 0 0 0; }
        h1 { margin: 0 0 16px 0; padding: 0 0 16px 0; font-size: 42px; font-weight: bold; letter-spacing: -2px; border-bottom: 1px solid #999; margin-right:0em;}
        h2 { font-size: 20px; margin: 0 0 6px 0; position: relative; }
        h2 span { position: absolute; bottom: 0; right: 0; font-style: italic; font-family: Georgia, Serif; font-size: 16px; color: #999; font-weight: normal; }
        p { margin: 0 0 16px 0; text-align:justify }
        a { color: #999; text-decoration: none; border-bottom: 1px dotted #999; }
        a:hover { border-bottom-style: solid; color: black; }
        ul { margin: 0 0 32px 17px; }
        #objective { width: 500px; float: left; }
        #objective p { font-family: Georgia, Serif; font-style: italic; color: #666; }
        dt { font-style: italic; font-weight: bold; font-size: 18px; text-align: right; padding: 0 26px 0 0; width: 150px; float: left; height: 100px; border-right: 1px solid #999;  }
        dd { width: 800px; float: right; }
        dd.clear { float: none; margin: 0; height: 15px; }
     </style>
</head>

<body>

    <div id="page-wrap">
    
        <!-- <img src="./static/profile_picPB.png" alt="My photo" height="200" id="pic"/> -->

        <div id="contact-info" class="vcard">
        
            <!-- Microformats! -->
        
            <h1 class="fn">L. Thibério Rangel, Ph.D.</h1>
            <h3>Computational Biologist</h3><br />
        
            <p>
                Email: <a class="email" href="mailto:lthiberiol@gmail.com">lthiberiol at gmail.com</a><br />
        <a href="https://github.com/lthiberiol/"><img src="./static/GitHub-Mark-120px-plus.png" height="33" style="margin-right:1em;" border="0"></a>
		<a href="https://twitter.com/thiberio" target="_blank"><img src="./static/twitter-2.jpg" height="35" style="margin-right:1em;" border="0"></a>
        <a href="https://scholar.google.com.br/citations?user=37VDh4gAAAAJ" target="_blank"><img src="./static/Black-Google-Scholar-Icon.jpg" height="33" style="margin-right:1em;" border="0"></a>
        <a href="https://www.linkedin.com/in/thiberio-rangel/" target="_blank"><img src="./static/linkedin_black_icon.png" height="33" style="margin-right:1em;" border="0"></a>
        <a href="https://lattes.cnpq.br/0660919183473128" target="_blank"><img src="./static/lattes_icon.png" height="33" border="0"></a>
            </p>
        </div>
                
        <div id="objective" style="width:1000px">
            <p style="text-indent: 1em;">
                Passionate and articulate Computational Biologist Ph.D. with sharp critical thinking and experience in multi-omics data analysis. Formulate drug discovery research by associating gene mutations/evolution with clinical metadata/outcome. Led collaborative efforts across multidisciplinary teams, adept at driving technical advancements, communicating complex concepts to diverse stakeholders.
                </p>
                
            <!-- Creative Commons Licence (by-nc-nd). See worditout.com for details -->
            <div style="width:auto;height:auto;margin-top: -1em;"><!-- You may use this wrapping div to restrict the height or width -->
                <script async src="https://worditout.com/word-cloud/5514932/embed.js"></script>
                <noscript>
            <p style="text-align:center;font-size:xx-small;overflow:auto;height:100%;"><a href="https://worditout.com/word-cloud/5514932" title="Click to go to this word cloud on WordItOut.com">&quot;thiberio - word cloud&quot;</a><br /></p>
                </noscript>
            </div>

        </div>
        
        <div class="clear"></div>
        
        <dl>
            <dd class="clear"></dd>

            <dt>Experience</dt>
            <dd>                   
                <h2>Sr. Scientist, Computational Biology</h2>
                <p><strong>Institution:</strong> <a href="https://gritstonebio.com/platforms/" target="_blank">Gritstone Bio, Boston, MA, USA</a><br />
                   <strong>Period:</strong> Nov 2022 <strong>-</strong> Fev 2024<br />
                    <strong>Key workstreams:</strong>
                    <ul style="margin-top:-1em">
                        <li>Main developer and analyst on discovery of Dark Matter shared neoantigens (e.g., Alt. splicing, nuORFs);</li>
                        <li>Headed cancer resistance mechanisms efforts through implementations of Loss of Heterozygosity of HLA alleles, Cancer Cell Fraction estimates, and Chromosome Instability methods;</li>
                        <li>Supported immunopeptidomics validation by compiling candidates with high HLA-biding probability;</li>
                        <li>Supported validation of novel experimental methods by developing experiment-specific QC methods;</li>
                        <li>Increased Gritstone’s sample collection by developing automated public database ingestion methods (e.g., RECOUNT3, TCGA, GTEx, and CCLE);</li>
                    </ul>
                </p>

                <h2>Research Scientist II, Computational Biology</h2>
                <p><strong>Institution:</strong> <a href="https://www.finchtherapeutics.com/platform/" target="_blank">Finch Therapeutics, Somerville, MA, USA</a><br />
                   <strong>Period:</strong> Jul 2021 <strong>-</strong> Nov 2022<br />
                    <strong>Key workstreams:</strong>
                    <ul style="margin-top:-1em">
                        <li>Designed and analyzed microbial function and phenotype experiments by collaborating with bench scientists;</li>
                        <li>Boosted stakeholder knowledge by communicating highly technical data clearly to a variety of audiences, including clinicians, bench scientists, industry executives, and academic researchers;</li>
                        <li>Positioned Finch as leader in microbiome therapeutics by innovating drug development platform technology and driving IP strategy in collaboration with legal partners;</li>
                        <li>Strengthened commercially viable therapeutics discovery platform by functioning as lead data analyst on a multifunctional team;</li>
                        <li>Increased external partners understanding by approving and assisting in the documentation of reports;</li>
                    </ul>
                </p>
                   
                <h2>Postdoctoral Associate</h2>
                <p><strong>Institution:</strong> Massachusetts Institute of Technology (MIT), Cambridge, MA, USA<br />
                    <strong>Supervisor:</strong> <a href="http://fournierlab.scripts.mit.edu/web/" target="_blank">Greg Fournier</a><br />
                    <strong>Period:</strong> Apr 2018 <strong>-</strong> Jun 2021<br />
                    <strong>Key workstreams:</strong>
                    <ul style="margin-top:-1em">
                        <li>Non-phylogenetic identification of gene co-evolution networks;</li>
                        <li>Modeling of among site substitution rate heterogeneity;</li>
                        <li>Application of Horizontal Gene Transfers to expand time calibration constraints (bayesian molecular clocks);</li>
                        </ul>
                    </p>

                <h2>Postdoctoral Fellow</h2>
                <p><strong>Simons Foundation Fellow</strong><br />
                    <strong>Institution:</strong> Universidade de São Paulo (USP), São Paulo, São Paulo, Brazil<br />
                    <strong>Supervisor:</strong> <a href="http://www.iq.usp.br/setubal/">João Carlos Setubal</a><br />
                    <strong>Period:</strong> Sep 2017 <strong>-</strong> Mar 2018<br />
                    <ul style="margin-top:-1em">
                        <li>Application of Horizontal Gene Transfers to expand time calibration constraints (bayesian molecular clocks);</li>
                    </ul>
                    </p>
            </dd>
            
            <dd class="clear"></dd>
            
            <dt>Education</dt>
            <dd>
                <h2>Ph.D. in Bioinformatics</h2>
                <p><strong>Institution:</strong> Universidade de São Paulo (USP), São Paulo, São Paulo, Brazil<br />
                    <strong>Title:</strong> Phylogenetic Networks and Horizontal Gene Transfer.<br />
                    <strong>Advisor:</strong> <a href="http://www.iq.usp.br/setubal/">João Carlos Setubal</a><br />
                    <strong>Period:</strong> Jul 2012 <strong>-</strong> Aug 2017<br />
                    <strong>Visiting researcher:</strong> one year in <a href="http://web.uconn.edu/gogarten/">Peter Gogarten's lab</a> at University of Connecticut as part of my Ph.D. research <strong>(from 04/2015 to 03/2016)</strong><br />
                    <strong>Fellowship: </strong><a href="http://www.fapesp.br/">FAPESP</a></p>

                <h2>M.Sc. in Host-Pathogen-Relationship Biology</h2>
                <p><strong>Institution:</strong> Universidade de São Paulo (USP), São Paulo, São Paulo, Brazil<br />
                    <strong>Title:</strong> Development of EGene platform for functional annotation and database integration: application and validation on transcript sequences of <i>Eimeria</i> spp. of domestic fowl.<br />
                    <strong>Advisor:</strong> <a href="http://lattes.cnpq.br/8644261025719801">Arthur Gruber</a><br />
                    <strong>Period:</strong> Sep 2009 <strong>-</strong> Jan 2012<br/>
                    <strong>Fellowship:</strong><a href="http://www.fapesp.br/">FAPESP</a></p>

                <h2>B.Sc. in Biological Sciences</h2>
                <p><strong>Institution:</strong> Universidade Federal da Paraíba (UFPB), João Pessoa, Paraíba, Brazil<br />
                    <strong>Title:</strong> Development of an online tool, GIAnT, for genomic island prediction on bacteria.<br />
                    <strong>Advisor:</strong> <a href="http://lattes.cnpq.br/4795833304329411">Demetrius Antônio Machado de Araujo</a><br />
                    <strong>Period:</strong> Jul 2005 <strong>-</strong> Apr 2009<br />
                    <strong>Fellowship:</strong><a href="http://www.cnpq.br/">CNPq</a></p>             
            </dd>
            <dd class="clear"></dd>

           <dt>Supplementary education</dt>
           <dd>
               <h2>Summer School on Networks and Evolution</h2>
               <p>Roscoff, Fr<br />
           Jul 3 - Jul 9, 2016</p>
                
           	<h2>Workshop on Molecular Evolution</h2>
               <p>Marine Biological Laboratory, Woods Hole, MA, USA<br />
		Jul 27 - Aug 6, 2014<br />
		url: <a href="https://molevol.mbl.edu">https://molevol.mbl.edu</a></p>
		
               <h2>5th Annual EuPathDB Workshop</h2>
               <p>University of Georgia, Athens, GA, USA<br />
		Jun 6 - Jun 9, 2010<br />
		url: <a href="http://workshop.eupathdb.org">http://workshop.eupathdb.org</a></p>
           </dd>
            
           <dd class="clear"></dd>

            <dt>Developed tools (a.k.a Portfolio)</dt>
            <dd>
                <p><strong><a href="https://github.com/lthiberiol/evolSimIndex">Python package for assessing Evolutionary Similarity Indices between gene families</a></strong><br />
                <strong>Description:</strong> Jupyter Notebook and Python package for assessing evolutionary similarities between gene families.</p>
                <p><strong><a href="http://www.coccidia.icb.usp.br/eimeriatdb">The <i>Eimeria</i> Transcript Database</a></strong><br />
                <strong>Description:</strong> web interface to access transcript database. Allows the user to interact with transcripts through BLAST and/or keyword searches. Developed as part of masters thesis together with automatic protein annotation components (related publications listed bellow). Written in Perl(backend and frontend) and PHP (frontend).</p>
                <p><strong><a href="http://www.bioinfo.ufpb.br/giant">GIAnT - <u>G</u>enomic <u>I</u>sland <u>An</u>alyzer <u>T</u>ool</a></strong><br />
                <strong>Description:</strong> web tool to predict genomic island through bias in genome content. Six different composition metrics implemented. Developed as part of undergraduate research. Written in Python (<a href="https://cherrypy.org/">CherryPy</a>).</p>
            </dd>
            
            <dd class="clear"></dd>
            
            <dt>Technical skills</dt>
            <dd>
                <p><strong>Statistics:</strong> Strong knowledge of statiscal methods applied to bioinformatics.</p>

                <h2>Programming languanges:</h2>
                    <ul>
                    <li><strong>Python</strong></li>
                        <ul style="margin-bottom:-0.1em;">
                        <li><i>well versed in several data analysis libraries (e.g. Pandas, SciPy, NumPy, Scikit-Learn, Plot.ly, and BioPython)</i></li>
                        </ul>
                    <li><strong>Perl</strong></li>
                    <li><strong>R</strong></li>
                    </ul>

                <h2>Cloud environments:</h2>
                    <ul>
                    <li><strong>AWS</strong></li>
                    <li><strong>GCP</strong></li>
                    </ul>

             
                <h2>Database</h2>
                    <ul>
                    <li><strong>MySQL</strong></li>
                    <li><strong>BigQuery</strong></li>
                    </ul>
            </dd>            
            <dd class="clear"></dd>
            
            <dt>Publications</dt>
            <dd>
                <h2>2022</h2>
                <p>Sarah L Schwartz, Lily Momper, <strong>Luiz Thiberio Rangel</strong>, Cara Magnabosco, Jan P Amend, Gregory P Fournier. <a/href="https://doi.org/10.1002/mbo3.1258">Novel nitrite reductase domain structure suggests a chimeric denitrification repertoire in the phylum Chloroflexi.</a> <i>MicrobiologyOpen</i>. 2022.<br />
                </p>
                <h2>2021</h2>
                <p><strong>L. Thiberio Rangel</strong>, Shannon M. Soucy, João Carlos Setubal, Gregory P. Fournier. <a/href="https://doi.org/10.1093/gbe/evab187">An Efficient, Nonphylogenetic Method for Detecting Genes Sharing Evolutionary Signals in Phylogenomic Data Sets.</a> <i>Genome Biology and Evolution</i>. 2021.<br />
                </p>
                <p>GP Fournier, KR Moore, <strong>LT Rangel</strong>, JG Payette, L Momper, T Bosak. <a/href="https://doi.org/10.1098/rspb.2021.0675">The Archean origin of oxygenic photosynthesis and extant cyanobacterial lineages.</a> <i>Proceedings of the Royal Society B</i>. 2021.<br />
                </p>
                <h2>2020</h2>
                <p>Taylor Wade, <strong>L. Thiberio Rangel</strong>, Soumya Kundu, Gregory P. Fournier, Mukul S. Bansal. <a/href="https://doi.org/10.1371/journal.pone.0232950">Assessing the Accuracy of Phylogenetic Rooting Methods on Prokaryotic Gene Families.</a> <i>PLOS One</i>. 2020.<br />
                </p>               
                <h2>2019</h2>
                <p><strong>Luiz Thiberio Rangel</strong>, Gregory P. Fournier. <a/href="https://doi.org/10.1101/835504">Fast-evolving alignment sites are highly informative for reconstructions of deep Tree of Life phylogenies.</a> <i>bioRxiv</i>. 2019; 835504.<br />
                </p>               
                <p><strong>Luiz Thiberio Rangel</strong>, Jeremiah Marden, Sophie Colston, João Carlos Setubal, Joerg Graf, Johann Peter Gogarten. <a/href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0214035">Identification and characterization of putative <i>Aeromonas</i> spp. T3SS effectors.</a> <i>PlosOne</i>. 2019 Jun 4; 14(6):e0214035<br />
                </p>
                <p>José SL Patané, Joaquim Martins, <strong>Luiz Thiberio Rangel</strong>, José Belasque, Luciano A Digiampietri, Agda Paula Facincani, Rafael Marini Ferreira, Fabrício José Jaciani, Yunzeng Zhang, Alessandro M Varani, Nalvo F Almeida, Nian Wang, Jesus A Ferro, Leandro M Moreira, João C Setubal. <a/href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6007-4">Origin and diversification of <i>Xanthomonas citri</i> subsp. citri pathotypes revealed by inclusive phylogenomic, dating, and biogeographic analyses.</a> <i>BMC genomics</i>. 2019 Sep 9; 20(1):700<br />
                </p>
                <h2>2017</h2>
                <p>Wang N, Pierson EA, Setubal JC, Xu J, Levy JG, Zhang Y, Li J, <strong>Rangel LT</strong>, Martins J Jr. <a/href="https://www.annualreviews.org/doi/abs/10.1146/annurev-phyto-080516-035513">The Candidatus Liberibacter-Host Interface: Insights into Pathogenesis Mechanisms and Disease Control.</a> <i>Annual Reviews Phytopathology</i>. 2017 Aug 4; 55:451-482<br />
                </p>
                <h2>2014</h2>
                <p>Wulff NA, Zhang S, Setubal JC, Almeida NF, Martins EC, Harakava R, Kumar D, <strong>Rangel LT</strong>, Foissac X, Bové JM, Gabriel DW. <a href="http://apsjournals.apsnet.org/doi/abs/10.1094/MPMI-09-13-0292-R">The Complete Genome Sequence of 'Candidatus <i>Liberibacter americanus</i>', Associated with Citrus Huanglongbing</a>. <i>Mol Plant Microbe Interact</i>. 2014 Feb; 27(2):163-76<br />
                </p>
                <h2>2013</h2>
                <p><strong>Rangel LT</strong>, Novaes J, Durham AM, Madeira AMBN, Gruber A. <a href="http://database.oxfordjournals.org/content/2013/bat006.abstract?keytype=ref&ijkey=Hr0IeeWM5lypiQe">The <i>Eimeria</i> Transcript DB: an integrated resource for annotated transcripts of protozoan parasites of the genus <i>Eimeria</i></a>. <i>Database (Oxford)</i>. 2013 Feb 14;2013:bat006<br />
                </p>
                <h2>2012</h2>
                <p>Novaes J<strong>*</strong>, <strong>Rangel LTLD*</strong>, Ferro M, Abe RY, Manha AP, de Mello JC, Varuzza L, Durham AM, Madeira AMBN, Gruber A. <a href="http://www.sciencedirect.com/science/article/pii/S002075191100275X">A comparative transcriptome analysis reveals expression profiles conserved across three <i>Eimeria</i> spp. of domestic fowl and associated with multiple developmental stages</a>. <i>International Journal of Parasitology</i>. 2012 Jan;42(1):39-48.<br />
                <strong>*Both authors contributed equally to the work.</strong></p>
            </dd>
            
            <dd class="clear"></dd>
            
  <!--          <dt>Posters</dt>-->
  <!--          <dd>-->
  <!--              <p>[9] <strong>Rangel LT</strong>, Setubal JC. Gene Transfer and Nitrogen Fixation Genes in Gamma- and Beta-Proteobacteria. RECOMB Comparative Genomics, 2013.</p>-->
  <!--              <p>[8] <strong>Rangel LTLD</strong>, Setubal JC, Wu F, Lin H e Mohamed N. Analysis of Proteobacterial genomes using Phylogenomic Networks. Brazilian Symposium on Bioinformátics, 2012.</p>-->
  <!--              <p>[7] Amgarten D, <strong>Rangel LTLD</strong>, Durham AM, Gruber A. Desenvolvimento de um Componente de anotação automática de vias metabólicas KEGG para a plataforma EGene. SIICUSP, 2010.</p>-->
		<!--<p>[6] <strong>Rangel LTLD</strong>, Durham AM,Gruber, A . Enriched annotation and functional assignment of proteins of <i>Eimeria</i> spp.: guilty by orthology. X-Meeting, 2010.</p>-->
		<!--<p>[5] Ferro M, Yamamoto R, <strong>Rangel LTLD</strong>, Durham AM, Gruber, A . EGene 2: a generic and modular platform for automated sequence processing and annotation. X-Meeting, 2009.</p>-->
		<!--<p>[4] <strong>Rangel LTLD</strong>, Rocha PKL, Paschoal AR, Vitorello CBM, Araújo DAM,  Rego TG. GIAnT: Genomic Island Analyzer Tool. X-Meeting, 2008.</p>-->
		<!--<p>[3] Rocha PKL, <strong>Rangel LTLD</strong>, Paschoal AR, Vitorello CBM, Araújo DAM, Rego TG. Quantitative analysis of genomic islands in Bacteria. X-Meeting, 2008.</p>-->
		<!--<p>[2] Rocha PKL, <strong>Rangel LTLD</strong>, Vitorello CBM, Araújo DAM, Rego TG. Identificação e caracterização de ilhas genômicas em bactérias do gênero <i>Mycoplasma</i>. Congresso Brasileiro de Genéttica, 2008.</p>-->
		<!--<p>[1] Bezerra AV, Martins AB, Souza HDN, <strong>Rangel LTLD</strong>, Rocha PKL, Araújo DAM. Avaliação do potencial leishmaniostático do composto natural isolado quercetina sobre <i>Leishmania chagasi</i>, Congresso Brasileiro de Parasitologia. 2007.</p>-->
  <!--          </dd>-->
            
  <!--          <dd class="clear"></dd>-->

            <dt>Hobbies</dt>
            <dd><p>Electric guitar</p>
	    </dd>
            
            <dd class="clear"></dd>
            
            <dt>References</dt>
            <dd><p>Available upon request</p></dd>
            
            <dd class="clear"></dd>
        </dl>
        
        <div class="clear"></div>
    
    </div>

</body>

</html>